src/redis-cli.c

```diff
@@ -303,14 +307,28 @@ static void cliPushHandler(void *, void *);
 
 uint16_t crc16(const char *buf, int len);
 
-static long long ustime(void) {
-    struct timeval tv;
-    long long ust;
+static long long nstime(void) {
+    struct timespec tv;
+    long long nst;
+#ifdef __MACH__ // OS X does not have clock_gettime, use clock_get_time
+    // https://gist.github.com/jbenet/1087739
```

Comment:
Mac has supported clock_gettime since OS 10.12, which has been EoL since 2019, I think it's safe to say that all modern users of Mac have clock_gettime.

Reply:
I didn't know that. Thank you very much.
I will remove that ifdef ~ endif in v2.